modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -164,7 +164,7 @@ class TopLevelNodes extends React.Component {
         if (!value) {
             return;
         }
-        value = `${value};\n`;
+        value = `${value}\n`;
```

Comment:
I think better to check if the semicolon is there and append based on that.

Reply:
semicolon will always be there @hevayo. It is getting attached from expression editor.